certified-connectors/HP Workforce Solutions/README.md

```diff
@@ -0,0 +1,24 @@
+
+## HP Workforce Solutions Connector
+
+HP Workforce Solutions provides a powerful and extensive REST API.  Using this API you can manage your devices, view incidents and much more! 
+
+## Prerequisites
+You will need the following to proceed:
+* An active subscription to HP Proactive Insights (A Product of HP Workfore Solutions)
+
+## Getting started
+- Start with creating a connection to the HP Workforce Solutions connector by logging in with your credentials to Proactive Insights.
+- Once a connection is created you can then run one of our Template Flows.
+
+## Supported Operations
+The connector supports the following operations:
+* `Hardware Inventory`: Gets the hardware inventory report
+* `TenantID`: Returns the tenantid for your account
+* `GetEvents`: Gets events
+* `GetUtilization`: Gets the utilization report
+
```

Comment:
Please add below sections as they are mandatory. 

1. Known issues and limitations
2. Deployment Instructions.

Reply:
We have updated the sections as requested, please check

Reply:
@vmanoharas Resolving due to this being done.